src/Components/Components/test/RendererTest.cs

```diff
@@ -2838,9 +3083,9 @@ public async Task CanTriggerEventHandlerDisposedInEarlierPendingBatchAsync()
             {
                 numEventsFired++;
 
-                // Replace the old event handler with a different one,
-                // (old the old handler ID will be disposed) then re-render.
-                component.OnTest = args => eventHandler(args);
+                    // Replace the old event handler with a different one,
+                    // (old the old handler ID will be disposed) then re-render.
+                    component.OnTest = args => eventHandler(args);
```

Comment:
Various issues with indentation here and in the file above. I guess VS is being weird.

Reply:
VS shennanigans, I've reformatted the code.